File: client/src/main/java/com/kotlarz/service/sensor/MockedTemperatureReader.java
package com.kotlarz.service.sensor;

import com.kotlarz.service.dto.TemperatureLog;

import java.util.Arrays;
import java.util.List;

public class MockedTemperatureReader implements TemperatureReader {
    @Override
    public List<TemperatureLog> readAll() {
        return Arrays.asList(TemperatureLog.mock());
    }

    @Override
    public void refreshDevices() {
    }
}

File: client/src/main/java/com/kotlarz/service/sensor/MockedTemperatureReader.kt
package com.kotlarz.service.sensor

import com.kotlarz.service.dto.TemperatureLog

import java.util.Arrays

class MockedTemperatureReader : TemperatureReader {
    override fun readAll(): List<TemperatureLog> {
        return Arrays.asList(TemperatureLog.mock())
    }

    override fun refreshDevices() {}
}
